package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "TODO-APP",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jasmine"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jalvar53/TODO-APP.git"
+  },
+  "dependencies": {
+    "core-js": "2.5.1",
+    "@angular/common": "5.0.2",
+    "@angular/core": "5.0.2",
+    "@angular/compiler": "5.0.2",
+    "@angular/forms": "5.0.2",
+    "@angular/platform-browser": "5.0.2",
+    "@angular/platform-browser-dynamic": "5.0.2",
+    "@angular/router": "5.0.2",
+    "@types/node": "8.0.53",
+    "rxjs": "5.5.2",
+    "zone.js": "0.8.18"
+  },
+  "devDependencies": {
+    "@angular/cli": "1.5.3",
+    "@angular/compiler-cli": "5.0.2",
+    "@angular/language-service": "5.0.2",
+    "jasmine": "2.8.0",
```

Comment:
You have dependencies that you are not using, like jasmine, angular, etc etc, remove it

Reply:
I added them on purpose because of the next stage of the development. You're right, is not need yet in the configuration.